src/components/BookList.jsx

```diff
@@ -0,0 +1,26 @@
+import BookElement from "./BookElement";
+import * as PropTypes from "prop-types";
+import { Component } from "react";
+
+class BookList extends Component {
+  render() {
+    let { books, deleteBook } = this.props;
+    const bookElements = books.map((book) => (
+      <BookElement key={book.id} book={book} deleteBook={deleteBook} />
+    ));
+
+    return (
+      <div>
+        <h2>Book List</h2>
+        {bookElements}
+      </div>
+    );
+  }
+}
```

Comment:
[OPTIONAL]
- [ ] Kindly make sure you write your class output correctly according to the standards. also
- [ ] You may consider using the functional component instead of class components as suggested in the core lats release of React

```suggestion
  render() {
    let { books, deleteBook } = this.props;

    return (
      <div>
        <h2>Book List</h2>
        {books.map((book) => (
      <BookElement key={book.id} book={book} deleteBook={deleteBook} />
    ));}
      </div>
    );
  }
}
```